### PR TITLE
Support for Helm 3.4.1 and k8s 1.19.4

### PR DESCRIPTION
**Pull request by gabrielcalderon**
* Added support for Helm 3.4.1 that was released [today](https://github.com/helm/helm/releases/tag/v3.4.1).
* Added support for kubectl 1.19.4 that was released [today](https://github.com/kubernetes/kubernetes/releases/tag/v1.19.4)

### CODE REVIEW

**Comment by gabrielcalderon** [comment]
@dtzar Got another update! :-)

**Comment by nnatter** [comment]
Is it planned to merge this in the near future? Thanks for your work! 🙂

**Comment by gabrielcalderon** [comment]
@dtzar Can you merge this, when you get some time. :-)

**Review by dtzar — APPROVED** [review]

**Review by dtzar — APPROVED** [review]

**Comment by dtzar** [comment]
@gabrielcalderon sorry for the delay I've been unplugged for months.  Now that you're a contributor, I trust you to approve and merge your own PR with simple version updates. Of course the build must pass.  Also minor preference would be to clean up or remove the squash merge comments before you do the merge.

If there are any desired additions, then yes I would definitely like to review the PR before merge.

**Comment by gabrielcalderon** [comment]
@leejones Thank you! I will make sure to squash the commits next time!

**Pull request merged**
